Documentation/tutorial/intro_toc.md

```diff
@@ -64,34 +64,60 @@ Property Name | Type              | Description
 ------------- | ----------------- | ---------------------------
 *name*        | string            | Specifies the title of the *TOC Item*.
 *href*        | string            | Specifies the hyperlink of the *TOC Item*.
-*homepage*    | string            | Specifies the homepage of the *TOC Item*. It is useful when *href* is linking to a folder.
-*uid*         | string            | Specifies the `uid` of the referenced file. If the value is set, it overwrites the value of *href*.
-*homepageUid* | string            | Specifies the `uid` of the homepage. If the value is set, it overwrites the value of *homepage*.
-*items*       | *TOC Item Object* | Specifies the children *TOC Items*s of current *TOC Item*.
+*items*       | *TOC Item Object* | Specifies the children *TOC Item*s of current *TOC Item*.
+
+**Advanced**: These properties is useful when a TOC links to or reference to another TOC, or links to a uid.
```

Comment:
Improved the document. Homepage is deprecated. href has 3 forms of use.

---
In reply to: [72650820](https://github.com/dotnet/docfx/pull/537#discussion_r72650820) [](ancestors = 72650820)